doc/api.apib

```diff
@@ -665,14 +669,8 @@ Resources related to inspect detailed status of Workflow Instances.
 
         {
             "active_states": [{
+                "latest_timestamp": 1536838486185,
```

Comment:
Should this not have the `initial_timestamp`?

Reply:
Unfortunately it won't. This is active state and we don't store initial_timestamp in datastore, and we probably don't want to read from bigtable to get it.
@danielnorberg and I talked about those missing timestamps in general. So that seems to be an improvement later.